### PR TITLE
Remove AccountPasswordStore and related flows

### PR DESCRIPTION
**Pull request by t3chguy**
As they are no longer needed since MSC3967

### CODE REVIEW

**Review by dbkr — APPROVED** [review]
Nice, although I'm afraid I've just merged my PR which changed a bunch of this stuff.

File: src/CreateCrossSigning.ts [hunk 1]
```diff
@@ -7,40 +7,13 @@ SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only
...
-        const canUploadKeysWithPasswordOnly = error.data.flows.some((f: UIAFlow) => {
-            return f.stages.length === 1 && f.stages[0] === "m.login.password";
-        });
-        return canUploadKeysWithPasswordOnly;
-    }
-}
-
 /**
  * Ensures that cross signing keys are created and uploaded for the user.
  * The homeserver may require user-interactive auth to upload the keys, in
```
Comment (richvdh) [inline src/CreateCrossSigning.ts:19]: looks like this comment is outdated?

File: src/CreateCrossSigning.ts [hunk 2]
```diff
@@ -69,19 +36,13 @@ export async function createCrossSigning(
...
-                password: accountPassword,
-            });
-        } else if (isTokenLogin) {
-            // We are hoping the grace period is active
+        try {
             await makeRequest({});
-        } else {
+        } catch (error) {
+            if (!(error instanceof MatrixError) || !error.data || !error.data.flows) {
+                throw error;
```
Comment (richvdh) [inline src/CreateCrossSigning.ts:43]:
```suggestion
                // Not a UIA response
                throw error;
```

**Review by richvdh — APPROVED** [review]
seems plausible, I guess

File: src/CreateCrossSigning.ts (hunk 1 above)
Comment (t3chguy) [reply src/CreateCrossSigning.ts:19]: Nope, the handling for that is still there, the server may require it, if it is old enough to not support that MSC/spec version

Comment (t3chguy) [reply src/CreateCrossSigning.ts:19]: Oh a later part of the comment is outdated though

Comment (richvdh) [reply src/CreateCrossSigning.ts:19]: yes, that's what I meant. GH wouldn't let me comment on it.

**Pull request merged**